.gitignore

```diff
@@ -1,3 +1,5 @@
+*.orig
+*.rej
```

Comment:
Those files are not generated by logrotate's build system.  Hence, in my opinion, they do not belong to logrotate's ```.gitignore```.  But the same holds for ```.*.swp``` or ```cscope.out```, which are already there.

@glensc, what is your opinion on this?